Materials/HotReloadTest/TestData/Shader_BlendingOff.txt

```diff
@@ -8,7 +8,7 @@
         }
     },
 
-    "BlendState": {
+    "GlobalTargetBlendState": {
```

Comment:
I think those are already in:
https://github.com/o3de/o3de-atom-sampleviewer/pull/463

Reply:
These changes are coming from merging ASV development branch. Not sure why it's still showing in the different.

Reply:
The merge seems working fine.